### PR TITLE
MarAsPaidController/Logging

### PR DESCRIPTION
**Pull request by Nasila**

### CODE REVIEW

File: .settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -19,350 +21,6 @@ org.eclipse.jdt.core.formatter.align_variable_declarations_on_columns=false
 org.eclipse.jdt.core.formatter.align_with_spaces=false
 org.eclipse.jdt.core.formatter.alignment_for_additive_operator=16
 org.eclipse.jdt.core.formatter.alignment_for_arguments_in_allocation_expression=16
-org.eclipse.jdt.core.formatter.alignment_for_arguments_in_annotation=0
```
Comment (magwas) [inline .settings/org.eclipse.jdt.core.prefs]: somehow your autoformat rules got deleted. ask help in mentoring to revert this change

**Comment by magwas** [comment]
updated branch, please pull before push

**Comment by szirbucz** [comment]
Updated branch, please pull before push

File: .settings/org.eclipse.jdt.core.prefs (hunk 1 above)
Comment (magwas) [reply .settings/org.eclipse.jdt.core.prefs:22]: still an issue

File: src/test/java/com/kodekonveyor/completion/MarkAsPaidControllerLoggingTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@TestedBehaviour("Logging")
+@TestedService("MarkAsPaidController")
+public class MarkAsPaidControllerLoggingTest
+    extends MarkAsPaidControllerStatusesTestBase {
+
+  @Test
+  @DisplayName("The name of the controller is logged")
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/completion/MarkAsPaidControllerLoggingTest.java:29]: There are a lot more test cases in this task. Call me to continue mentoring when it is convenient for you.

**Comment by szirbucz** [comment]
No reply from the coder in the past 3 weeks.
